go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/mainflux/mainflux
 
 go 1.19
 
+replace github.com/mainflux/mproxy => /home/sammy/Documents/mproxy
```

Comment:
The PR was merged so you can update the version of mproxy

Reply:
https://github.com/mainflux/mproxy/pull/36